rest-api/ext/follow.xqy

```diff
@@ -108,16 +67,17 @@ function follow:post(
     }
 
   (: Now insert the following info into the user's profile :)
-  let $insert-noop :=
-    if($profile/jbasic:user/jbasic:follows) then
-      xdmp:node-insert-child($profile/jbasic:user/jbasic:follows, $follow-object)
-    else
-      let $insert-node :=
-        element jbasic:follows {
-          attribute type { "array"},
-          $follow-object
-        }
-      return xdmp:node-insert-child($profile/jbasic:user, $insert-node)
+  let $profile := user:get($username)
+  let $follows := $profile/jbasic:json/jbasic:user/jbasic:follows
```

Comment:
user:get() now returns element(json:basic). you need to update every occurrence of:

$profile/jbasic:json/jbasic:user/jbasic:follows/...

with

$profile/jbasic:user/jbasic:follows/...